methods/concat/tests.js

```diff
@@ -0,0 +1,66 @@
+import test from 'ava';
+import concat from '.';
+
+test('returns the array itself', t => {
+    const ary = [1, 2, 3];
+    t.deepEqual(
```

Comment:
@aldnav I'm not sure if `deepEqual` checks the identity of the values (i.e. if they are the _same_ objects). I only know it to check if the arrays' items are the same. If it does, that's good. Otherwise, perhaps [`t.is`](https://github.com/avajs/ava/blob/master/docs/03-assertions.md#isvalue-expected-message) is what you need 🤔

Reply:
@arnellebalane I'll be more selective of ava assertions next time. Thanks for the tip. Updated here https://github.com/arnellebalane/rbjs/pull/40/commits/caffdb62e4e46f5742595d507997e480afded424